tests/features/joinup_document/document.moderation.feature

```diff
@@ -0,0 +1,114 @@
+@api
+Feature: Document moderation
+  In order to manage documents
+  As a user of the website
+  I need to be able to transit the documents from one state to another.
+
+  Background:
+    Given users:
+      | name            |
+      | Crab y Patties  |
+      | Gretchen Greene |
+      | Kirk Collier    |
+    And the following owner:
+      | name          |
+      | thisisanowner |
+    And the following collection:
+      | title             | The Naked Ashes                 |
+      | description       | The wolverine is a Marvel hero. |
+      | logo              | logo.png                        |
+      | banner            | banner.jpg                      |
+      | elibrary creation | registered users                |
+      | moderation        | no                              |
+      | state             | validated                       |
+      | owner             | thisisanowner                   |
+      | policy domain     | Demography and population       |
+    And the following collection user membership:
+      | collection      | user            | roles       |
+      | The Naked Ashes | Gretchen Greene | member      |
+      | The Naked Ashes | Kirk Collier    | facilitator |
+
+  Scenario: Available transitions change per eLibrary and moderation settings.
+    # For post moderated collections with eLibrary set to allow all users, even
+    # authenticated users can create content.
```

Comment:
> For post-moderated collections with eLibrary creation set to allow all users to create content,
authenticated users that are not members of the collection can create documents.